src/main/java/edu/northeastern/truthtree/controller/basicInfo/BasicInfo.java

```diff
@@ -0,0 +1,38 @@
+package edu.northeastern.truthtree.controller.basicInfo;
+
+import org.json.simple.JSONArray;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.northeastern.truthtree.service.basicInfo.BasicInfoService;
+import edu.northeastern.truthtree.service.basicInfo.IBasicInfoService;
+import edu.northeastern.truthtree.service.collections.CollectionsService;
+import edu.northeastern.truthtree.service.collections.ICollectionsService;
+
+@RestController
+public class BasicInfo implements IBasicInfo {
+
+	@Override
+	@RequestMapping("/api/states")
+	public String getBasicStatesInfo() {
+		IBasicInfoService service = new BasicInfoService();
```

Comment:
Consider using spring bean to create objects instead of using new. Consider making it a class variable. You can refer to the Attributes class.

Reply:
Are you referring to the AttributeResource? If so, I believe these are doing the same thing, just that the new structure is splitting the method into multiple methods that live in separate classes.